packages/ocaml-base-compiler/ocaml-base-compiler.4.03.0/opam

```diff
@@ -1,6 +1,7 @@
 opam-version: "2.0"
 synopsis: "Official 4.03.0 release"
 maintainer: "platform@lists.ocaml.org"
+license: "QPL-1.0 AND LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
```

Comment:
4.03 was the first release to be distributed under the LGPL with linkink exceptions and not QPL.

Reply:
are you sure? According to https://github.com/ocaml/ocaml/commit/7fc2d214c4ee8853615fc3e5649f45e11d85378d it was only done in 4.04.0. The 4.03 branch has it technically but it was only added after the release of 4.03.0

Reply:
Hm, and according to ocaml/ocaml@0ca4d6a74 the plan was to update the license for 4.03 . Keeping the license is probably more precise indeed. Sorry for the noise.

Reply:
I agree with @kit-ty-kate : 4.03.0 was released before the switch to LGPL.